app/controllers/api/v1/admin/communications_controller.rb

```diff
@@ -34,6 +35,19 @@ def destroy
 
         private
 
+        def update_params
+          params.require(:communication).permit(:title, :text, :published, :recurrent_on)
+        end
+
+        def handle_attachments
+          return unless params[:communication][:image]
+
+          @communication.image.purge
```

Comment:
We just need to do this if we want to destroy it, right?

Reply:
no, we need to do it always.
if we don't even if the files are detacched, they remain in S3 wich could be a problem